skyportal/tests/api/test_spectrum.py

```diff
@@ -819,3 +821,59 @@ def test_spectrum_plot_only_some_spectra(
     assert len(instruments) == 2
     assert 'LRIS' in str(instruments)
     assert 'SEDM' in str(instruments)
+
+
+def test_post_get_spectrum_type(upload_data_token, public_source, public_group, lris):
```

Comment:
Let's also add a test to ensure that providing an invalid type is handled properly.